opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java

```diff
@@ -148,10 +139,10 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
         if (servletRequest.getAttribute(SERVER_SPAN_CONTEXT) != null) {
             chain.doFilter(servletRequest, servletResponse);
         } else if (isTraced(httpRequest, httpResponse)){
-            SpanContext extractedContext = tracer.extract(Format.Builtin.HTTP_HEADERS,
+            SpanContext extractedContext = GlobalTracer.get().extract(Format.Builtin.HTTP_HEADERS,
                     new HttpServletRequestExtractAdapter(httpRequest));
 
-            final Span span = tracer.buildSpan(httpRequest.getMethod())
+            final Span span = GlobalTracer.get().buildSpan(httpRequest.getMethod())
```

Comment:
Is there a chance to have the `GlobalTracer#get` response from the first call to be different than this one? If so, what are the consequences to this call?